whoami/.env-dist

```diff
@@ -27,7 +27,7 @@ WHOAMI_HTTP_AUTH=
 WHOAMI_OAUTH2=no
 # In addition to Oauth2 authentication, you can configure basic authorization
 # by entering which authorization group can log into your app. You create
-# groups of email addresses in the `traefik` folder by running `makr groups`. 
+# groups of email addresses in the `traefik` folder by running `make groups`. 
 WHOAMI_AUTHORIZED_GROUP=
```

Comment:
I think this should be WHOAMI_OAUTH2_AUTHORIZED_GROUP

Reply:
Fixed.